indicoio/api/prebuilt.py

```diff
@@ -35,3 +35,33 @@ def pdf_extraction(self, data, job_results=False, **pdf_extract_options):
         else:
             job.wait()
             return job.result()
+
+    def document_extraction(self, data, job_results=False, **document_extraction_options):
+        if not isinstance(data, list):
+            raise IndicoInputError(
+                "This function expects a list input. If you have a single piece of data, please wrap it in a list"
+            )
+        data = [pdf_preprocess(datum) for datum in data]
+        data = json.dumps(data)
+        option_string = ",".join(
```

Comment:
this seems like a utility we should have available separately

Reply:
I think we need to rewrite this portion to handle it via variables, instead of inline into the query. Easier cause we just need a json dict of vars and no query manipulation.